components/org.wso2.carbon.identity.api.user.recovery/src/test/java/org/wso2/carbon/identity/recovery/endpoint/impl/CaptchaApiServiceImplTest.java

```diff
@@ -108,7 +108,7 @@ public void testGetCaptcha(boolean reCaptchaEnabled, String reCaptchaType,
         }
     }
 
-    public Properties getSampleConfigFile() throws IOException {
+    private Properties getSampleConfigFile() throws IOException {
```

Comment:
As this is a public repo we can't change a method from public to private

Reply:
Ack. It was updated due to https://github.com/wso2-extensions/identity-governance/pull/756#discussion_r1493807990

Change reverted